test-runner/README.md

```diff
@@ -64,11 +85,26 @@ test:
 
 ```bash
 python test-runner/test_runner.py -f path/to/tests.yaml
-VAR=world python test-runner/test_runner.py -f path/to/tests.yaml
+VAR1=world python test-runner/test_runner.py -f path/to/tests.yaml
 ```
 
 In the example above, the first output will be `hello`, and the second output will be `world`.
 
+Modifying the above example to use the `env` dictionary option:
+
+```yaml
+test:
+  cmd: echo "${VAR1:-hello}"
+  env:
+    VAR1: "Yo!"
+```
+
+```bash
+python test-runner/test_runner.py -f path/to/tests.yaml
+VAR1=world python test-runner/test_runner.py -f path/to/tests.yaml
+```
+In this instance, the default value of `hello` is overridden first by the value in the `env` variable dictionary, and then by the value passed on the test command line. Thus, the first output will be `Yo!`, and the second output will be `world`.
```

Comment:
you need another newline between these. You can't have text adjacent to code fences.